FirebaseCore.podspec

```diff
@@ -28,6 +28,7 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
   s.public_header_files = 'Firebase/Core/Public/*.h', 'Firebase/Core/Private/*.h'
   s.private_header_files = 'Firebase/Core/Private/*.h'
   s.framework = 'Foundation'
+  s.osx.framework = 'AppKit'
```

Comment:
Thanks!

It also needs 
```
s.ios.framework = 'UIKit'
s.tvos.framework = 'UIKit'
```

Reply:
Done (in alphabetical order, matching the order of `deployment_target` above).